dispatch/static/manager/src/js/components/ContentEditor/embeds/VideoEmbed.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+
+import { FormInput, TextInput } from '../../inputs'
+
+class VideoEmbedComponent extends React.Component {
+
+  constructor(props){
```

Comment:
As a convention, we like to leave a space before the opening bracket of a function:

```js
constructor(props) {
```

(Change this for the other methods too)